ocamltest/tests.ml

```diff
@@ -48,29 +48,25 @@ let test_of_action action =
   test_actions = [action]
 }
 
-let run_actions log testenv actions =
-  let total = List.length actions in
+let run_actions depth log testenv actions =
   let rec run_actions_aux action_number env = function
     | [] -> (Result.pass, env)
     | action::remaining_actions ->
       begin
-        Printf.fprintf log "Running action %d/%d (%s)\n%!"
-          action_number total (Actions.name action);
+        Printf.fprintf log "%s %s\n%!"
+          (String.make depth '*') (Actions.name action);
         let (result, env') = Actions.run log env action in
-        Printf.fprintf log "Action %d/%d (%s) %s\n%!"
-          action_number total (Actions.name action)
-          (Result.string_of_result result);
```

Comment:
Here you are removing information from the logic: `string_of_result` is not shown anymore. I am not sure that is a good idea; it sounds useful in the log to be able to quickly locate actions that failed, and to know if an action succeeded or was skiped. (Or is the idea that this would already be clear from the `Action.run` logging?)

Reply:
OK, I missed that I think. But yes, the plan is to show the command that executed and its output when there is a failure, which would be more informative.